src/pages/AddLiquidity/index.tsx

```diff
@@ -137,13 +137,12 @@ export default function AddLiquidity({
 
     const deadlineFromNow = Math.ceil(Date.now() / 1000) + deadline
 
-    let estimate,
-      method: (...args: any) => Promise<TransactionResponse>,
+    let method: (...args: any) => Promise<TransactionResponse>,
       args: Array<string | string[] | number>,
       value: BigNumber | null
     if (currencyA === ETHER || currencyB === ETHER) {
       const tokenBIsETH = currencyB === ETHER
-      estimate = router.estimateGas.addLiquidityETH
+      // estimate = router.estimateGas.addLiquidityETH
```

Comment:
why this is commented?

Reply:
it was causing some errors, i can look further into it

Reply:
better to check this.But less important than the UI fixes in telegram